requirements.txt

```diff
@@ -2,4 +2,5 @@ networkx>=2.2.0
 numpy>=1.10.0
 scipy>=1.0.0
 scikit-learn>=0.18.2
+Cython==0.29.6
```

Comment:
Out of curiosity, why `==` instead of `>=`? I just want to get this merged already so we can play with it, so don't change it, but just wondering.